src/Compilers/CSharp/Portable/Errors/ErrorFacts.cs

```diff
@@ -204,6 +204,10 @@ internal static int GetWarningLevel(ErrorCode code)
 
             switch (code)
             {
+                case ErrorCode.WRN_AutoPropertyAllowNull:
```

Comment:
Currently the warning will probably not working with warnaserror:nullable? Is this intended?

Reply:
It's not intended. This should probably be suppressed along with any other nullability warnings.